src/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './enum-xyz.js'
+export * from './deprecated.js'
```

Comment:
@texastoland Does this need to be exposed? In terms of other use cases, I was thinking we'd more just make a markdown file for each example use case we think up with a few words about them and then link them off of the readme.

Reply:
If `deprecated` isn't exposed it's a breaking change in which case no need to have it. Or I misunderstand?

Reply:
I think the Markdown is referring to something else right?

Reply:
Resolving to continue in the main thread in context of `ActionCreators` too 👇🏼